kubectl-pg/cmd/util.go

```diff
@@ -26,6 +26,11 @@ func getConfig()(*restclient.Config){
 }
 
 func getCurrentNamespace() (string){
-	currentNamespace, _ := exec.Command("kubectl", "config", "view", "--minify", "--output", "jsonpath={..namespace}").CombinedOutput()
-	return string(currentNamespace)
+	namespace, _ := exec.Command("kubectl", "config", "view", "--minify", "--output", "jsonpath={..namespace}").CombinedOutput()
+	currentNamespace := string(namespace)
+	if currentNamespace == "" {
+		currentNamespace = "default"
+		return currentNamespace
```

Comment:
this return is obsolete :)
the final return is enough

Reply:
I didn't much attention here. Sorry, will change it. :)